integrations/pgvector/src/haystack_integrations/components/retrievers/pgvector/embedding_retriever.py

```diff
@@ -75,7 +76,9 @@ def __init__(
             and the most similar documents are the ones with the smallest score.
             **Important**: if the document store is using the `"hnsw"` search strategy, the vector function
             should match the one utilized during index creation to take advantage of the index.
-
+        :param filter_policy: Policy to determine how filters are applied. Defaults to "replace".
```

Comment:
```suggestion
        :param filter_policy: Policy to determine how filters are applied.
```

Lately, we have stopped mentioning defaults in docstrings, if they can be easily inferred from the signature.